assets/src/dashboard/parts/connect/index.js

```diff
@@ -187,6 +199,14 @@ const ConnectLayout = () => {
 							dangerouslySetInnerHTML={ { __html: optimoleDashboardApp.strings.account_needed_subtitle_2 } }
 						/>
 					</div>
+
+					<div className="flex py-3">
```

Comment:
We can also add a `items-center` class on all three items in this list so that the checkmark is centered. Maybe also map them to an array with the contents we can loop through and output them so we don't have duplicated markup.

Reply:
@abaicus fixed all suggestions